doc/design/README.md

```diff
@@ -1,8 +1,39 @@
 # Design Documents
 
-In here you can find design documents to all constructs of iceoryx. At the 
+In here you can find design documents to all constructs of iceoryx. At the
 moment those design documents are neither complete nor final.
 
-In [draft](./draft/) are design drafts of current features contained which are under 
-implementation. The goal is to provide other developers a first idea on how 
+In [draft](./draft/) are design drafts of current features contained which are under
+implementation. The goal is to provide other developers a first idea on how
 the feature can be used and how it will be implemented.
+
+## Add diagrams using plantuml
```

Comment:
Wikipedia writes it as [PlantUML](https://en.wikipedia.org/wiki/PlantUML). You could think of using the same style.